### DISCUSSION TITLE
Possible typo in docs\OpperatorKernels.md

### DISCUSSION THREAD

**Discussion by georgen117** [microsoft/onnxruntime#11774] [Other Q&A] [answered]
I was looking at the recent work to to add LongformerAttention to the code and found Longformer

https://github.com/microsoft/onnxruntime/blob/master/docs/OperatorKernels.md

| Op Name | Parameters | OpSet Version | Types Supported |
|---------|------------|---------------|-----------------|
|LongformerAttention|*in* input:**T**<br> *in* weight:**T**<br> *in* bias:**T**<br> *in* mask:**T**<br> *in* global_weight:**T**<br> *in* global_bias:**T**<br> *in* global:**G**<br> *out* output:**T**|1+|**T** = tensor(float), tensor(float16)|

The global:**G** seems to be a typo should the **G** be a **T**?

Since the top of the document indicates the file is automatically generated I was wondering if the **G** is indeed a typo does it exist some place else?

**Comment by ytaous** [comment, marked as answer]
doesn't look like a typo, it's just type alias, see:
https://github.com/microsoft/onnxruntime/blob/master/docs/ContribOperators.md#com.microsoft.LongformerAttention

**Reply by georgen117** [reply]
> doesn't look like a typo, it's just type alias, see: https://github.com/microsoft/onnxruntime/blob/master/docs/ContribOperators.md#com.microsoft.LongformerAttention

Agreed: G is its own type. 

```
Type Constraints
T : tensor(float), tensor(float16)
Constrain input and output types to float tensors.
G : tensor(int32)
Constrain to integer types
```

I think where I was confused was that 'G' was not in the Types supported column. It should probably be updated. So I guess there is still a typo of sorts. It is lacking one of the "Type Supported" values
